.github/workflows/main.yml

```diff
@@ -10,23 +10,21 @@ jobs:
     strategy:
       fail-fast: false
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
       with:
         python-version: 3.11
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        if [ -f update/requirements.txt ]; then pip install -r update/requirements.txt; fi
+        pip install -r update/requirements.txt
     - name: Run Update Script
       working-directory: ./update
       run: |
         python update_symbols.py
     - name: commit
       run: |
-        git config user.name "GitHub Actions Bot"
-        git config user.email "<>"
         git add .
-        git commit -m "Update Symbols"
+        git diff --staged --quiet || git -c user.name='GitHub Actions Bot' -c user.email='<>' commit -m 'Update Symbols'
```

Comment:
Wow that was quick. See [this discussion post](https://github.com/orgs/community/discussions/26560) as a hint about using github bot credentials in commits made during CI runs.